salt/modules/zypperpkg.py

```diff
@@ -1387,10 +1393,15 @@ def refresh_db(force=None, root=None):
     salt.utils.pkg.clear_rtag(__opts__)
     ret = {}
     refresh_opts = ["refresh"]
+    global_opts = []
     if force is None:
         force = __pillar__.get("zypper", {}).get("refreshdb_force", True)
     if force:
         refresh_opts.append("--force")
+    if kwargs.get('gpgautoimport', False):
+        global_opts.append('--gpg-auto-import-keys')
```

Comment:
Not sure if splitting global opts makes huge sense in this case, if you want to split global opt out, maybe better to avoid using `refresh_opts = global_opts + refresh_opts`, but put global opts to the call this way `__zypper__(root=root).refreshable.call(*global_opts, *refresh_opts)`

Reply:
I changed it to `__zypper__(root=root).refreshable.call(*global_opts, *refresh_opts)`